app/controllers/customers_controller.rb

```diff
@@ -37,6 +37,46 @@ def create
     end
   end
 
+  def add_customer
+    if is_customer_exist(params[:customer][:phone_number])
+      @customer = Customer.find_by(phone_number: params[:customer][:phone_number])
+    else
+      @customer = Customer.new(customer_params)
+      @customer.save
+    end
+  end
+
+  def is_customer_exist(customers_phone_number)
+    Customer.find_by('phone_number = ?',customers_phone_number)
+  end
+
+  def assign_cleaner(customer_id,customers_city_id,customers_date)
+    cleaners = City.find(customers_city_id).cleaner.where(email_confirmed: true)
```

Comment:
It is firing INNER JOIN as below

SELECT "cleaners".* FROM "cleaners" INNER JOIN "cleaners_cities" ON "cleaners"."id" = "cleaners_cities"."cleaner_id" WHERE "cleaners_cities"."city_id" = ? AND "cleaners"."email_confirmed" = ?  [["city_id", 2], ["email_confirmed", "t"]]

how can i improve it???